src/KnitUtil.kt

```diff
@@ -19,4 +20,12 @@ fun <T : LineNumberReader> KnitContext.withLineNumberReader(file: File, factory:
     return reader
 }
 
-operator fun File.div(path: String): File = File(this, path.replace("/", File.separator))
\ No newline at end of file
+operator fun File.div(path: String): File = File(this, path.replace("/", File.separator))
+
+internal fun String.capitalizeFirstChar() = this.replaceFirstChar {
```

Comment:
This is the officially proposed substitution for the deprecated `.capitalize()`. Seems too bulky to copy-paste